python_src/src/lamp_py/performance_manager/pipeline.py

```diff
@@ -46,13 +49,29 @@ def parse_args(args: List[str]) -> argparse.Namespace:
     return parser.parse_args(args)
 
 
+def legacy_metadata_exists(rpm_db_manager: DatabaseManager) -> bool:
+    """
+    are there any records in the legacy metadata table
+    """
+    metadata = rpm_db_manager.select_as_dataframe(
+        sa.select(LegacyMetadataLog.path)
+    )
+
+    return not metadata.empty
```

Comment:
I don't think this is really required. If everything is the performance manager pipeline is now configured to read from the new MetadataLog table, whether or not the LegacyMetadataLog exists shouldn't matter.

Reply:
i was worried about some deployment error on our part and wanted to protect against that. i had planned on taking this out on a subsequent PR where everything is complete.